README.md

```diff
@@ -138,7 +138,7 @@ pods = runpod.get_pods()
 pod = runpod.get_pod(pod.id)
 
 # Create a pod
-pod = runpod.create_pod("test", "runpod/stack", "NVIDIA GeForce RTX 3070")
+pod = runpod.create_pod("test", "runpod/stack", "NVIDIA RTX A6000")
```

Comment:
What exactly is the point in changing this? It offers no benefit and does not fix any typos or misinformation.  PRs should only be created if you have something meaningful to contribute.